pep-0680.rst

```diff
@@ -100,69 +114,74 @@ Maintenance Implications
 Stability of TOML
 -----------------
 
-The release of TOML v1 in January 2021 indicates stability. Empirically, TOML
-has proven to be a stable format even prior to the release of TOML v1. From the
-`changelog <https://github.com/toml-lang/toml/blob/master/CHANGELOG.md>`_, we
-see TOML has had no major changes since April 2020 and has had two releases in
-the last five years.
+The release of TOML 1.0 in January 2021 indicates the TOML format should
```

Comment:
```suggestion
The release of TOML 1.0.0 in January 2021 indicates the TOML format should
```

The release is actually 1.0.0. Not sure how important detail this is. Also if we want to refer to the major version (>=1,<2) then maybe use v1?

Thanks for the PR, by the way, this is great! :smiley:

Reply:
I was just drawing on my experience for which sounded the most idiomatic, what was generally used on the TOML spec repo, and the fact that "TOML 1.0" indicated approximately the correct degree of version precision for most usages, but you're right that it isn't strictly correct. To address this, I changed usages (like this one) that referred to the specific version to `1.0.0`, and to the 1.x major version series in general to `1.x`.

@pradyunsg any further input here?

Reply:
What you've done sounds about right to me.

I'll likely be cutting 1.0.1 at some point this year, so... there's that. :)